src/Altinn.App.Api/Controllers/ResourceController.cs

```diff
@@ -250,5 +250,19 @@ public ActionResult GetRuleConfiguration(string org, string app, string id)
 
             return new FileContentResult(fileContent, MimeTypeMap.GetMimeType(".json"));
         }
+
+        /// <summary>
+        /// Get the footer layout
+        /// </summary>
+        /// <param name="org">The application owner short name</param>
+        /// <param name="app">The application name</param>
+        /// <returns>The footer layout in the form of a string.</returns>
+        [HttpGet]
+        [Route("{org}/{app}/api/footer")]
+        public ActionResult GetFooterLayout(string org, string app)
+        {
+            string layout = _appResourceService.GetFooter();
+            return Ok(layout);
```

Comment:
```suggestion
            if (layout is null)
            {
                return NoContent()
            }
            
            return Ok(layout);
```

Reply:
I think it already sends a `204` when returning `Ok(null)`.

Reply:
Maybe? It would make kind of sense.

Reply:
It doesn't return a 204 for `Ok(null)`.